deta/service.py

```diff
@@ -5,10 +5,19 @@
 import struct
 import typing
 import urllib.error
+from pathlib import Path
 
 JSON_MIME = "application/json"
 
 
+class CustomEncoder(json.JSONEncoder):
```

Comment:
I'd like to suggest a specific custom class name to avoid ambiguity✌🏿  
```suggestion
class CustomJSONEncoder(json.JSONEncoder):
```

Reply:
Thanks for review, I've already committed fix.